config/runtime.exs

```diff
@@ -0,0 +1,170 @@
+import Config
+
+# config/runtime.exs is executed for all environments, including
+# during releases. It is executed after compilation and before the
+# system starts, so it is typically used to load production configuration
+# and secrets from environment variables or elsewhere. Do not define
+# any compile-time configuration in here, as it won't be applied.
+# The block below contains prod specific runtime configuration.
+
+# ## Sensitive values should be passed through the environment. An example
+# EnvironmentFile that can be used with systemd is under
+# documents/prod_files/example-environment-file
+
+# ## Using releases
+#
+# If you use `mix release`, you need to explicitly enable the server
+# by passing the PHX_SERVER=true when you start it:
+#
+#     PHX_SERVER=true bin/teiserver start
+#
+# Alternatively, you can use `mix phx.gen.release` to generate a `bin/server`
+# script that automatically sets the env var above.
+if Teiserver.ConfigHelpers.get_env("PHX_SERVER", nil) do
```

Comment:
Why is this needed? I don't follow

Reply:
This is a standard env var expected by phoenix, the framework used by teiserver. If it's not set, there won't be any actual http server start up.
I don't think we should modify that one, or prefix it with `TEI_` since the tooling like `mix release` will generate scripts that expects or set this specific env variable.
That should be transparent from your side though, if you start with `teiserver/bin/teiserver start` this handled automatically.